benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm

```diff
@@ -0,0 +1,181 @@
+# off. The recovery is driven by the accumulated elastic stresses and thus
+# provides a basic test for the viscoelasticity implementation. 
+#
+# Compositional fields are used to track the viscoelastic stresses and material 
+# representing the beam. To improve the accuracy of tracking the beam interface,
+# a discontinuous discretization and bound preserving limiter is used for the 
+# compositional fields. Significantly, the time step is limited to 1 Kyr through
+# the "Maximum time step" parameter and a relatively high (0.5) CFL value. Using
+# a constant time step ensures the effective (viscoelastic) viscosity of the
+# viscoelastic beam and viscous fluid remains constants throughout the model run.
+#
+# As currently constructed, the model will run for 50 Kyr with a gravitational
+# acceleration of 10 m/s^2. To produce unbending of the beam after the model
+# has finished, change the parameter "End time" to 500 Kyr and set the 
+# gravitational acceleration to 0 m/s^2. A restart file is written every 10 Kyr 
+# and the parameter "Resume computation = auto" specifies that a model should  
+# restart from a checkpoint file if one is present.     
+
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 50e3
+set Use years in output instead of seconds = true
+set Resume computation                     = auto
+set Use direct solver for Stokes system    = false
+set Linear solver tolerance                = 1e-7
+set Nonlinear solver scheme                = IMPES
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 1
+set Number of cheap Stokes solver steps    = 2000
+set CFL number                             = 0.5
+set Maximum time step                      = 1e3
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = surface
+set Surface pressure                       = 0.
+
+# Model geometry (7.5x5 km, 0.025 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 75
+    set Y repetitions = 50
+    set X extent      = 7.5e3
+    set Y extent      = 5e3
+  end
+end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Boundary classifications
+subsection Model settings
+  set Fixed composition boundary indicators   = bottom, top, right  
+  set Fixed temperature boundary indicators   = bottom, top, left, right
+  set Prescribed velocity boundary indicators =  
+  set Free surface boundary indicators        = 
+  set Tangential velocity boundary indicators = bottom, top, right
+  set Zero velocity boundary indicators       = left
+  set Enable elasticity                       = true
```

Comment:
I would like to remove this subsection in #2105. Would you be ok with moving the parameter into the `Formulation` subsection that handles all modifications to the equations?